spec/integration/cli/process_cmd_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'gooddata'
+require_relative '../../lcm/integration/support/connection_helper'
+
+describe 'GoodData::CLI - process commands', :vcr do
+  describe '#create' do
+    before(:all) do
+      env = LcmConnectionHelper.environment
+      @client = LcmConnectionHelper.production_server_connection
+      @credential_params =  "-U #{env[:username]} -P #{env[:password]} -t #{env[:gd_project_token]} -s https://#{env[:prod_server]} -l --no-verify-ssl -p jidh28wupkbxueq9j5006jd31nsj7nze"
+    end
+
+    after(:all) do
+      if @new_process_uri
+        @client.delete @new_process_uri
+      end
+    end
+
+    describe '#from_path' do
```

Comment:
Neither `from_path` nor `as_component` are documented in the CLI help.

```
$ gooddata process create -h
NAME
    process - Manage processes (ETLs, Ruby bricks..)

SYNOPSIS
    gooddata [global options] process create

COMMANDS
    create - Create a new process
```

Reply:
I am not sure why. Try `gooddata process create` with no arguments.

Reply:
Ammended a fix. `gooddata process create` now returns a full help with explanations..